src/DependencyInjection/Configuration.php

```diff
@@ -8,6 +8,11 @@
 
 final class Configuration implements ConfigurationInterface
 {
+    private array $strategies = [
```

Comment:
I this this suits better here
```suggestion
    private const STRATEGIES = [
```

Reply:
fixed